# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mplayer/jack.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Well, the usual fixes for big endian systems ...

  - Rene Rebe <rene@exactcode.de>

--- MPlayer-1.0pre6a/libao2/ao_jack.c	2004-12-07 03:24:15.000000000 +0100
+++ MPlayer-1.0pre6a-fixed/libao2/ao_jack.c	2005-03-08 15:22:38.000000000 +0100
@@ -153,7 +153,11 @@
 			m = 1;
 			break;
 		default:
+#ifndef __BIG_ENDIAN__
 			format = AFMT_S16_LE;
+#else
+			format = AFMT_S16_BE;
+#endif
 			bits_per_sample = 16;
 			m = 2;
 			break;
